packages/mask/src/plugins/Savings/SNSAdaptor/SavingsDialog.tsx

```diff
@@ -97,12 +103,46 @@ export function SavingsDialog({ open, onClose }: SavingsDialogProps) {
         chainId,
     )
 
+    const { value: yfiTokens } = useAsync(async () => {
```

Comment:
Move hooks for `aaveTokens` and  `yfiTokens` to `packages/mask/src/plugins/Savings/hooks`. It's too many stuff for a single file.

```ts
// hooks/useAAVETokens.ts
export function useAAVETokens() {
    return []
}
```

```ts
// hooks/useYFITokens.ts
export function useYFITokens() {
    return []
}
```